metrics/config.go

```diff
@@ -32,3 +33,10 @@ type M3Config struct {
 	Service  string `yaml:"service"`
 	Env      string `yaml:"env"`
 }
+
+// PrometheusConfig defines prometheus configuration.
+type PrometheusConfig struct {
+	ListenAddress string `yaml:"listen_address"`
+	// HandlerPath if not define use default /metrics.
+	HandlerPath string `yaml:"handler_path"`
```

Comment:
The yaml config keys are camel cased in the tally package:

https://github.com/uber-go/tally/blob/f266f90e9c4d5894364039a324a05d061f2f34e2/prometheus/config.go#L33-L58

Perhaps we should prefer camel case here as well so that it's not a breaking change if we want to swap `PrometheusConfig` out with the tally `prometheus.Configuration` type in the future.